### ISSUE TITLE
[Support]: Use bundled version instead of installing package

### ISSUE THREAD

**Issue by kamoii** [progfolio/elpaca#47]
### Emacs Version

GNU Emacs 30.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.36, cairo version 1.17.6) of 2022-12-25

### Operating System

Arch

### question

`consult-dir` packages depends on `project` package. So when installing `consult-dir`:
```
(elpaca consult-dir)
```
elpaca also clone and build `project` package as well.

Recent Emacs bundles `project.el`. Is there a way to tell elpaca to not install `project` package and use the bundled one?
I'm thinking of something like:
```
(elpaca (project :use-bundled t))
```

### Confirmation

- [X] I've read the documentation and I still am unsure.

**Comment by progfolio** [comment]
project.el appears have been introduced in the following Emacs commit:

```
commit f8c720b55b9419c849ea9febe6f888761a61949b
Author: Dmitry Gutov <dgutov@yandex.ru>
Date:   Fri Jul 10 04:34:41 2015 +0300

    Introduce a Project API
    
    * lisp/progmodes/project.el: New file.
    
    * lisp/cedet/ede.el: (project-try-ede): New function.
    (project-root): New implementation.
    
    * lisp/progmodes/elisp-mode.el (emacs-lisp-mode):
    Set project-search-path-function.
    (elisp--xref-find-references): Delegate some logic to
    project-search-path.
    (elisp-search-path): New function.
    (elisp-xref-find): Don't implement `matches' anymore.
    
    * lisp/progmodes/etags.el: Don't implement `matches'.
    Delegate some logic to project-search-path.
```

Which is included in emacs-25.0.9:

```
$ git tag --contains f8c720b55b9419c849ea9febe6f888761a61949b | head -n 1
emacs-25.0.90
```

I've added project to `elpaca-ignored-dependencies` since it is guaranteed to be present in the Emacs versions Elpaca supports.

Consideirng consult-dir supports a min Emacs version of 26.1, it may be worth filing a request with the maintainer to drop the project dependency declaration from the Pakcage-Requires metadata:

https://github.com/karthink/consult-dir/blob/ed8f0874d26f10f5c5b181ab9f2cf4107df8a0eb/consult-dir.el#L7

**Comment by kamoii** [comment]
Thanks for your quick response!

`elpaca-delete`ed `consult-dir` and `project`. `elpaca-update`ed elpaca. Restarted emacs and confirmed `project` is not installed. `consult-dir` works with no problem too.

> In general, the elpaca-ignored-dependencies variable can be pushed to if you run into this sort of problem again.

Thanks for the info. `elpaca-ignored-dependencies` is what I was looking for.

> Considering consult-dir supports a min Emacs version of 26.1, it may be worth filing a request with the maintainer to drop the project dependency declaration from the Package-Requires metadata:

Done!
